fiftyone/core/ontology.py

```
@@ -193,3 +272,99 @@ def from_dict(cls, d: dict) -> "AnnotationOntology":
+    Args:
+        name: the ontology name
+
+    Returns:
+        an :class:`Ontology`
+
+    Raises:
+        ValueError: if no ontology with the given name exists
+    """
+    doc = _objects_by_slug(name).order_by("-version").first()
+    if doc is None:
+        raise ValueError(f"Ontology '{name}' not found")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)